src/docs/files/devguide08.md

```diff
@@ -0,0 +1,62 @@
+### How to Participate as an Auctioneer using the Gnosis Safe App
+
+#### Connect Wallet & App
+
+1. First, you will need to have a [Gnosis Safe](https://gnosis-safe.io/) that you control deployed in Mainnet.
```

Comment:
I think we should mention that the app is also available on rinkeby. But this guide focuses on mainnet